docs/introduction.mdx

```diff
@@ -61,6 +61,8 @@ The ["@emotion/core"](https://www.npmjs.com/package/@emotion/core) package requi
 
 - Best when using React with a build environment that can be configured.
 
+- Also works well even if using React in a nonconfigurable build environment like an unejected `Create React App` app. This is true for the current version of `Create React App` as of October 2019. This method and the one below both work out of the box for the default and `--typescript` configurations of a new `Create React App` app.
```

Comment:
I think I would just merge this with the previous bullet point, smth like:
```
Best when using React with a build environment that can be configured, but also works...
```

And I'm not sure how the second part of this point:
> This is true for the current version of `Create React App` as of October 2019. This method and the one below both work out of the box for the default and `--typescript` configurations of a new `Create React App` app.

is relevant.